### ISSUE TITLE
AWX Does Not Update User Object Permissions When LDAP User Flags By Group Is Removed

### ISSUE THREAD

**Issue by falcon78921** [ansible/awx#5051]
##### ISSUE TYPE
 - Feature Request

##### SUMMARY

A user was wondering how AWX handles a change in LDAP Team Mapping (e.g. If the `is_superuser` mapping is removed, does AWX update the user objects?).

I observed the following:

If the `LDAP User Flags by Group` is removed, pre-existing users in AWX do not update according to the changes made. 

##### STEPS TO REPRODUCE

1. Create a LDAP connection from AWX.
2. Specify a `LDAP User Flags by Group` parameter.
3. Login with a LDAP user and verify its creation within AWX.
4. Verify the user has the proper permissions.
5. Change the `LDAP User Flags by Group` to no longer reflect the user.

##### EXPECTED RESULTS

I would expect the user object within AWX to update itself based on the LDAP mapping specified.

##### ACTUAL RESULTS

It looks like the user object has to be manually updated in order for the permissions to update. If `is_superuser` is granted to many users in a LDAP group and the mapping has to change, this could be an undesired behavior.

**Comment by alebeta90** [comment]
Hi,

I had been configuring LDAP access and logged in with a user before setting `LDAP User Flags by Group` and then afterwards I decided to activate the option to flag by group, and the user is not getting the `is_superuser` permission.

My configuration looks like this

``` yaml
{
 "is_superuser": [
  "ou=Company-Admin,dc=example,dc=internal"
 ]
}
```

Don't know why I get the [], when I see Tower documnetation it does not suppose to appear. Don't know if it is normal.

I tried deleting and adding the user again and still the same issue is happening.

thanks
